src/common/__tests__/log-search-store.test.ts

```diff
@@ -0,0 +1,82 @@
+/**
```

Comment:
This file for a technical test could be located right next to implementation. We could have \_\_tests\_\_ for more behavioural stuff. Superficial.